guides/common/modules/ref_troubleshooting-discovery.adoc

```diff
@@ -3,33 +3,42 @@
 
 If a machine is not listed in the {ProjectWebUI} in *Hosts* > *Discovered Hosts*, inspect the following configuration areas to help isolate the error:
 
-* In the {ProjectWebUI}, navigate to *Hosts* > *Templates* > *Provisioning Templates*.
-. Click *Build PXE Default* to redeploy the default PXELinux template.
-* Verify the `pxelinux.cfg/default` configuration file on the TFTP {SmartProxy}.
+* Ensure that the prerequisites are met.
+For more information, see xref:prerequisites-for-using-discovery_{context}[].
+* Ensure you have set Discovery for booting and built the PXE boot configuration files.
+For more information, see xref:setting-discovery-as-the-default-PXE-boot-option_{context}[].
+* Verify the following configuration files on the TFTP {SmartProxy}:
```

Comment:
How do you verify them? Or do you only check that they're present?

Reply:
They must be present and they must have `discovery` as the default boot option.
That's the minimum.
I'll add that.